studio-docs/source/explorer/connecting-authenticating.mdx

```diff
@@ -182,6 +182,17 @@ The body of the `POST` request that's sent to the configured GraphQL endpoint fo
 </td>
 </tr>
 
+<tr>
+<td>
+
+##### `explorer.CryptoJS`
+</td>
+<td>
+
+This exposes the crypto-js package, refer to the crypto-js docs for documentation on how to use it. https://www.npmjs.com/package/crypto-js
```

Comment:
Suggested replacement:

This exposes the `crypto-js` package for use within your preflight script. For available functions, [see the documentation](https://www.npmjs.com/package/crypto-js).